### ISSUE TITLE
math.py interferes with standard math module

### ISSUE THREAD

**Issue by aeshthetic** [strinking/mawabot#38]
the file mawabot/mawabot/cogs/general/math.py interferes with modules that depend on the `math` module found in the python standard library

**Comment by raylu** [comment]
this shouldn't be an issue in python3? can you describe the problem in more detail?

**Comment by emmiegit** [comment]
It shouldn't interfere, since the two imports are different.
```py
import math # built-in library
from .math import a, b, c # math.py
```

**Comment by aeshthetic** [comment]
An issue occurred where the requests module attempted to import `math` and failed to do so within the directory, until I renamed `math.py` to `mmath.py` in order to test whether it was interfering with the import, and that's what seemed to be causing the issue. 
![image](https://user-images.githubusercontent.com/23178243/28755394-f3e83c34-7527-11e7-92b1-82f5d2e484ce.png)
![image](https://user-images.githubusercontent.com/23178243/28755401-046a5484-7528-11e7-9112-28ffc82ea9dc.png)

**Comment by emmiegit** [comment]
Oh well. Can you rename `math.py` to `calc.py` (and adjust the header comment and stuff appropriately)?
Thanks Abhijay.

**Comment by aeshthetic** [comment]
alright; thanks

**Comment by raylu** [comment]
I don't see requests in the traceback (nor do I think it's compatible with asyncio anyway). can you paste the relevant code and traceback instead of screenshots?

**Comment by emmiegit** [comment]
Fixed in 0f183a1
